Example/iOS/Views/AnimatedSwitchRow.swift

```diff
@@ -0,0 +1,119 @@
+// Created by Cal Stephens on 1/4/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import Epoxy
+import Lottie
+import UIKit
+
+// MARK: - AnimatedSwitchRow
+
+final class AnimatedSwitchRow: UIView, EpoxyableView {
+
+  // MARK: Lifecycle
+
+  init() {
+    super.init(frame: .zero)
+    translatesAutoresizingMaskIntoConstraints = false
+    layoutMargins = UIEdgeInsets(top: 16, left: 24, bottom: 16, right: 24)
+    group.install(in: self)
+    group.constrainToMarginsWithHighPriorityBottom()
+    backgroundColor = .systemBackground
+  }
+
+  required init?(coder _: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: Internal
+
+  struct Content: Equatable {
+    var animationName: String
+    var title: String
+    var onTimeRange: (from: CGFloat, to: CGFloat)
+    var offTimeRange: (from: CGFloat, to: CGFloat)
```

Comment:
How about using `ClosedRange<CGFloat>` instead? Will that also allow us to remove our equatable implementation?

Reply:
Good idea!